DeveloperConsole/InitializePatch.cs

```diff
@@ -1,20 +1,17 @@
 ﻿using HarmonyLib;
 using Il2Cpp;
-using Il2CppTLD.AddressableAssets;
 using UnityEngine;
-using UnityEngine.ResourceManagement.AsyncOperations;
+using UnityEngine.AddressableAssets;
 
-namespace DeveloperConsole {
+namespace DeveloperConsole
+{
 
-    [HarmonyPatch(typeof(BootUpdate), "Start")]
+	[HarmonyPatch(typeof(BootUpdate), "Start")]
```

Comment:
Indentation

Reply:
my VS was auto formatting.. turned it off.